palanaeum/static/palanaeum/js/page_nav.js

```diff
@@ -49,4 +49,14 @@ $(function(){
             $("#desktop-header-small:visible").fadeOut("fast");
         }
     });
-});
\ No newline at end of file
+});
+
+function show_hidden_pages(elipsis_element) {
```

Comment:
Nitpick: ellipsis is written with two l :)

Reply:
Right you are! I looked up the right spelling later but this one slipped through.